Tools/autotest/common.py

```diff
@@ -815,9 +812,9 @@ def do_set_mode_via_command_long(self, mode, timeout=30):
                          )
             m = self.mav.recv_match(type='HEARTBEAT',
                                     blocking=True,
-                                    timeout=remaining)
```

Comment:
Was this change intentional ?

Reply:
Yes.  (By inspection) it seems you could get a "heartbeat not received" error message when the real problem is that you're out of time.